src/WSRequestHandler_Sources.cpp

```diff
@@ -1679,6 +1679,40 @@ RpcResponse WSRequestHandler::SetAudioMonitorType(const RpcRequest& request)
 	return request.success();
 }
 
+/**
+* Get the default settings for a given source type.
+*
+* @param {String} `sourceKind` Source name.
+*
+* @api requests
+* @name GetSourceDefaultSettings
+* @category sources
+* @since 4.9.0
+*/
+RpcResponse WSRequestHandler::GetSourceDefaultSettings(const RpcRequest& request)
+{
+	if (!request.hasField("sourceKind")) {
+		return request.failed("missing request parameters");
+	}
+
+    QString sourceKind = obs_data_get_string(request.parameters(), "sourceKind");
+
```

Comment:
nitpick: remove this blank line to group the `if` statement with the variable it checks